i18n/ja.toml

```diff
@@ -5,7 +5,10 @@ other = "Home"
 other = "Back to home"
 
 [flip]
-other = "フリップ"
+other = "Flip it!"
```

Comment:
Hi @enkaito, thanks for the updates. Due to I'm unfamiliar with Japanese. Does this change mean most Japanese people will use English to represent this flip behavior?

Reply:
Or do you mean this flip behavior is poorly translated, so English will be used for now?